package.json

```diff
@@ -47,16 +47,17 @@
 		"@prismicio/mock": "0.0.10",
 		"@prismicio/types": "^0.1.27",
 		"@rollup/plugin-inject": "^4.0.4",
-		"@sveltejs/kit": "1.0.0-next.328",
-		"@typescript-eslint/eslint-plugin": "^5.23.0",
-		"@typescript-eslint/parser": "^5.23.0",
+		"@sveltejs/kit": "^1.0.0-next.587",
```

Comment:
Since the commits in this PR were made, `@sveltejs/kit` has been updated to `v1.7.0`. Could we update all dependencies to the latest versions?

I like using `npx npm-upgrade` to quickly update everything, but any method works. 🙂

Reply:
Looks good. `npx npm-upgrade` timed out for me a few times, so I just manually upgraded this one.